arrows/core/compute_ref_homography_core.cxx

```diff
@@ -518,7 +521,7 @@ ::estimate( frame_id_t frame_number,
       // current_frame).
       if( (ti.active && ti.ref_id != earliest_ref) || ti.ref_id == frame_number )
       {
-        ti.ref_loc = output->homography()->map( ti.ref_loc );
+        ti.ref_loc = output->homography()->map( fts->feature->loc() );
```

Comment:
Is the change here just a better way to access the current location? Or was it a switch from projecting the reference location to the current location?

Reply:
I perhaps should have copied the commit message bodies into the description. Here's the relevant one:

> If `ti` describes an active track whose reference frame is not the current frame, using the reference location is a mistake because `output->homography()` maps *current-frame* coordinates to reference-frame coordinates.  `fts->feature->loc()` provides current information.

Reply:
Thanks.